recipe/meta.yaml

```diff
@@ -66,15 +65,21 @@ requirements:
     - tomli >=1.1.0
     - tomli-w
     - httpx-ws >=0.6.0
+    - aiosqlite >=0.20.0
+    - uv >=0.5
+    - questionary >=2.0.1
+    #- a2wsgi >=1.10.7 # NA on conda-forge
```

Comment:
Is it a mandatory dependency?

Reply:
Appears so - its listed in depdencies
https://github.com/bentoml/BentoML/blob/main/pyproject.toml#L70
    "a2wsgi>=1.10.7",

Reply:
yes @ocefpaf , 
I added a2wsgi last week - https://github.com/conda-forge/staged-recipes/pull/28716
its available now for use - I was trying to get kantoku added too 
- https://github.com/conda-forge/staged-recipes/pull/28715
https://github.com/bentoml/BentoML/blob/main/pyproject.toml#L31

My intent was to get both kantoku & a2wsgi built and available, before fixing and merging this feedstock